pkg/detectors/websitepulse/websitepulse.go

```diff
@@ -0,0 +1,85 @@
+package websitepulse
+
+import (
+	"context"
+	"net/http"
+	"regexp"
+	"strings"
+	"io/ioutil"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"websitepulse"}) + `\b([0-9a-z]{32})\b`)
```

Comment:
suggestion: we can make this a bit stricter as the keys are in hex encoding so we can only work with `a-f`